module6/src/main/java/Main.java

```diff
@@ -0,0 +1,128 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.Writer;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Main {
+    private final static String DIR_PREFIX = "|----";
+    private final static String FILE_PREFIX = "|   ";
+    private final static String DEFAULT_INDENTATION = "   ";
+
+    public static void main(String[] args) {
```

Comment:
Please create classes to make api more clear.
E.g. class for directories and files calculation, class for result file analisys etc.